### DISCUSSION TITLE
Ray was unable to bind to the remote server's port (on Siteground)

### DISCUSSION THREAD

**Discussion by web-elements** [spatie/ray#808] [Q&A]
**Describe the bug**
When I try to connect to any Siteground server using ssh key
"Ray was unable to bind to the remote server's port.
Is the port in use or is someone else using ray on this server?"
But I can connect correctly through terminal on mac.
I have seen https://github.com/spatie/ray/issues/685 and https://unix.stackexchange.com/questions/427189/how-to-cleanup-ssh-reverse-tunnel-socket-after-connection-closed
but I don't have permission to do this thinks on a siteground server.

**Versions**
Ray version (you can see this in "About Ray"): 2.4.0

**To Reproduce**
1. add a siteground server
2. insert the data in the form (the same that I use in the terminal where is working)
3. try to connect to the server

**Expected behavior**
It should connect to server.

**Desktop (please complete the following information):**
 - OS: macOS
 - Version 12.6
 
 **Additional context**
I haven't had any problems with other servers, only with those of Siteground

**Comment by jkbennemann** [comment]
I do have the same issue with other hosters as well, not just siteground.
I think this could be related to some kind of version incompatibilites.

The servers I tested run on
- OpenSSH v.8.9p1
- OpenSSL 3.0.2

**Comment by colinr-l** [comment]
I have the same problem as other posters when trying to connect Ray to a Wordpress site on a remote server at Siteground.
I am using Windows and am able to connect to remote server via ssh on Windows Powershell but get "Ray was unable to bind to the remote server's port" when trying to connect using Ray:

![image](https://github.com/spatie/ray/assets/98536341/22416422-2fa8-4c3d-bb31-01bf525ab77e)

**Comment by menshinobi** [comment]
Any solution to this issue?

**Comment by lansezzz** [comment]
Any solution to this issue?

**Comment by federicociti** [comment]
Any solution to this issue?

**Comment by colinr-l** [comment]
No, unfortunately not, I reverted to manual coding debug statements.
